lib/pages/home/home.dart

```diff
@@ -63,6 +63,11 @@ class HomePage extends StatelessWidget {
         },
       ),
       drawer: isWiderScreen ? null : SideDrawer(),
+      onDrawerChanged: (isOpened) {
+        if (!isOpened) {
```

Comment:
Can we invert the condition to `isClosed` ?

Reply:
I don't understand your request, `isOpened` is a parameter from [DrawerCallback.](https://api.flutter.dev/flutter/material/DrawerCallback.html) in [Scaffold](https://api.flutter.dev/flutter/material/Scaffold-class.html). So why changing it ?

Reply:
My bad. I missed it.